t/features/export.t

```diff
@@ -0,0 +1,57 @@
+use v6;
+use Test;
+use _007::Test;
+
+{
+    my $program = q:to/./;
+        export func greet(name) {
+            say("Good evening, Mr " ~ name);
+        }
+        greet("Bond");
+        .
+
+    outputs $program, "Good evening, Mr Bond\n", "export syntax works for functions";
+}
+
+{
+    my $program = q:to/./;
+        export macro moo() {
+            say("Macro!");
+        }
+        moo();
+        .
+
+    outputs $program, "Macro!\n", "export syntax works for macros";
+}
+
+{
+    my $program = q:to/./;
+        export my name = "Bond";
+
+        say(name);
+        .
+
+    outputs $program, "Bond\n", "export syntax works for 'my' variables";
+}
+
+{
+    my $program = q:to/./;
+        export 2 + my name = "Bond";
+        .
+
+    parse-error $program,
+        X::Export::Nothing,
+        "export syntax is not allowed if 'my' variable is not on the left";
+}
+
+{
+    my $program = q:to/./;
+        export "but there was nothing there to export";
+        .
+
+    parse-error $program,
+        X::Export::Nothing,
+        "export syntax is not allowed if the 'my' variable is missing";
+}
+
+done-testing;
```

Comment:
Now... As I said, perhaps I missunderstand something. But it seem to me - glancing at the code - that infix, postfix and more is included - IS exportable - but in that case, where is the tests? This only focus on my and func? Further, it seem to me that the tests is more comprehensive for my than func (...)? Why? Yes, that may be because of the comment but... hmm... in that case why the added lines with infix (...) ?